app/components/appMainModule/committeePageModule/committeePageRoutes.js

```diff
@@ -44,7 +44,7 @@ function committeePageRoutes($stateProvider) {
     template: require('./committeePage.html'),
     ncyBreadcrumb: {
       label: '{{ committee.name }}',
-      parent: 'appMain.locality'
+      parent: 'appMain'
```

Comment:
Interesting. So this fixes the breadcrumbs in one way, but breaks it in another. Before this change, the breadcrumbs looked correct if you navigated to the page through the site, but were blank if you directly loaded the url. Now, there's a breadcrumb if you load the page directly, but I think it's broken if you navigate to the page through the site? Or that could be all wrong. Breadcrumbs, huh.